spec.bs

```diff
@@ -4231,7 +4231,7 @@ metrics collected from a single execution of a worklet function:
 <dl dfn-for="execution metrics">
   : <dfn>code fetch time averager</dfn>
   :: An [=averager=].
-  : <dfn>script timeout occurred</dfn>
+  : <dfn>script timed out</dfn>
   :: A [=boolean=], initially false.

```

Comment:
nit: may be useful to add a sentence to explain that it's only the single script's timeout, not affected by cumulative timeout.

Reply:
Done, though it's probably not true of this CL (I think my CL for cumulative metrics I supposed to fix it, but I am not 100% sure it does, it's hard to read off the diff).